test/e2e/framework.go

```diff
@@ -200,6 +214,14 @@ func (f *Framework) afterEach() {
 	f.Client = nil
 }
 
+func (f *Framework) CreateNamespace(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
+	ns, err := createTestingNS(baseName, c, labels)
```

Comment:
It is used in a few places NOT using Framework.  There's value to having layers of behavior, i think.